atlasdb-commons/src/test/java/com/palantir/exception/SafeSSLPeerUnverifiedExceptionTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * (c) Copyright 2022 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.exception;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.palantir.logsafe.UnsafeArg;
+import org.junit.Test;
+
+public class SafeSSLPeerUnverifiedExceptionTest {
+
+    private static final String badArgKey = "bad-arg";
+    private static final String badArgValue = "iamdangerous!";
+    private static final UnsafeArg<String> unsafeArg = UnsafeArg.of(badArgKey, badArgValue);
+
+    @Test
+    public void logMessage_doesNotHaveUnsafeArgs() {
```

Comment:
nit: this really is a legacy way of naming the tests. Most of our new test names are just camel cased.

Reply:
Also, will the message ever contain safe args. Basically the idea is you test happy and non-happy paths of the same case.

Reply:
`nit: this really is a legacy way of naming the tests. Most of our new test names are just camel cased.` Why is this considered legacy? I'm happy to conform to camelcase for test names, but have a preference for this style of test naming as it much clearer what you're actually testing.

Reply:
🤔 i suppose if these were static helper methods, it would make it more readable to prefix test with method name you are actually testing. 
Otherwise, I would think, the tests are generally testing behaviour of that class and no particular method in that class (Of course there are many exceptions to this rule, and we certainly test at method level sometimes).
Also, I think we want to test a small functionality in a test and a short name usually is sufficient and is generally readable.